python/graphstorm/model/loss_func.py

```diff
@@ -412,29 +545,32 @@ def forward(self, pos_score, neg_score):
 class LinkPredictContrastiveLossFunc(GSLayer):
     r""" Contrastive Loss function for link prediction.
 
-        The positive and negative scores are computed through a
-        score function as:
+    The positive and negative scores are computed through a
+    score function as:
+
+        .. math::
 
             score = f(<src, rel, dst>)
 
-        And we treat a score as the distance between `src` and
-        `dst` nodes under relation `rel`.
+    And we treat a score as the distance between `src` and
+    `dst` nodes under relation `rel`.
 
-        In contrastive loss, we assume one positive pair <src, dst>
-        has K corresponding negative pairs <src, neg_dst1>,
-        <src, neg_dst2> .... <src, neg_dstk> When we compute the
-        loss of <src, dst>, we follow the following equation:
+    In contrastive loss, we assume one positive pair <src, dst>
+    has K corresponding negative pairs <src, neg_dst1>,
+    <src, neg_dst2> .... <src, neg_dstk> When we compute the
+    loss of <src, dst>, we follow the following equation:
+
+        .. math::
 
-            .. math::
             loss = -log(exp(pos\_score)/\sum_{i=0}^N exp(score_i))
 
-        where score includes both positive score of <src, dst> and
-        negative scores of <src, neg_dst0>, ... <src, neg_dstk>
+    where score includes both positive score of <src, dst> and
+    negative scores of <src, neg_dst0>, ... <src, neg_dstk>
```

Comment:
```suggestion
    negative scores of <src, neg_dst0>, ... <src, neg_dstk>.
```

Reply:
Is the ``N`` in the equation same as the ``k`` of neg edge pairs? If so, better to change ``N`` to ``k``.